### ISSUE TITLE
Funds Have Not Arrived From Exchange 

### ISSUE THREAD

**Issue by elev83328** [adityapk00/zecwallet-lite#185]
Please make sure you have the latest ZecWallet. 

**Describe the bug**
When withdrawing/sending crypto to Zecwallet Lite, transaction is not showing in balance, said amount of coins have yet to arrive, this is way outside of the usual transaction speed. 

I’ve discussed with Kraken, they have assured me that these coins are showing as sent and received from their end of things. They have sent me the links to Zecblock explorer, showing this transaction as being received correctly. 

I briefly asked Zooko his opinion, he sent me in the direction of the wallet. 

I would love some help with this. Read some past similar threads. Can’t solve from my end at the moment. 

**To Reproduce**
1. Open Wallet 
2. See error

**Expected behavior**
To see my transaction sent from exchange having arrived to my wallet.

**Desktop (please complete the following information):**

**Additional context**

**Comment by adityapk00** [comment]
The Zcash blockchain has been increasing in size suddenly, and so wallets are having a bit of an issue right now.

1. Did you use a transparent address? If so, you can use another wallet, restore the seed and use it like that.

**Comment by elev83328** [comment]
So essentially delete the application. Redownload the application. And use my seed and password to enter the wallet again ?

**Comment by elev83328** [comment]
@adityapk00 yes. i used t-address. i’ve done what you have said. uninstalled and reinstalled the app. and restored the seed. it’s now only showing the latest transaction, and not my entire amount.

currently trying to rescan the wallet and be patient. the wallet seems to have trouble rescanning.

**Comment by Ildep** [comment]
#193
